plugins/janus_videoroom.c

```diff
@@ -3445,6 +3445,8 @@ static json_t *janus_videoroom_process_synchronous_request(janus_videoroom_sessi
 				janus_config_add(config, c, janus_config_item_create("is_private", "yes"));
 			if(videoroom->require_pvtid)
 				janus_config_add(config, c, janus_config_item_create("require_pvtid", "yes"));
+			if(videoroom->signed_tokens)
+				janus_config_add(config, c, janus_config_item_create("signed_tokens", "yes"));
```

Comment:
Shouldn't this be in the "edit" part too? Even if "edit" doesn't expose a way to modify that property, if you don't have lines to save the value there, a permanent "edit" will result in not saving the `signed_tokens` property, which would be lost as a consequence.

Reply:
Yeh, you're right. I forgot about `edit`.

Reply:
@lminiero done